b2sdk/http_constants.py

```diff
@@ -12,6 +12,8 @@
 # These constants are needed in different modules, so they are stored in this module, that
 # imports nothing, thus avoiding circular imports
 
+LIST_FILE_NAMES_MAX_LIMIT = 10000  # https://www.backblaze.com/b2/docs/b2_list_file_names.html
```

Comment:
But why is this exposed at all? It should be just some internal limitation of b2 and user should just iterate over pages seamlessly. And it should not be used by ourselves as well since there is always `nextFileName` field if there is next page.

Reply:
tl;dr: because we use this stuff in CLI

and longer answer:
what you wrote is exactly why. Because only nice interface for `ls` makes it so "seamless"  that if we want to limit requests made, we need this number to be known in CLI
https://github.com/Backblaze/B2_Command_Line_Tool/blob/a478fb0e55c9274fdb2911f26b2006c00e890e66/b2/_cli/argcompleters.py#L35C1-L52

Reply:
I will use const in there as you suggested it is cheap and just makes sense.

IDK if adding manual pagination is worth it; we already gave a lot of manual control there to the user through fetch_count, I wouldn't complicate API more by adding another param for use case that can be already achieved thru what we have already.